src/boot/efi/meson.build

```diff
@@ -376,6 +377,11 @@ foreach efi_elf_binary : efi_elf_binaries
         if name.startswith('linux')
                 boot_stubs += exe
         endif
+
+        # This is supposed to match exactly one time
+        if name == 'addon@0@.efi.stub'.format(efi_arch)
+                efi_addon = exe.full_path()
+        endif
```

Comment:
Please explain.

Reply:
You will find a `addonia32.efi.stub` and `addonx64.efi.stub` on x86 (if the toolchain supports it), so this won't match exactly once.

Reply:
But only one of them is `efi_arch`, no? The other is name constructed using `efi_alt_arch`.

Reply:
I suppose you're right. If that's what you want…

Reply:
Well, we just need one for tests. We're not doing anything serious with the file. We may add more tests in the future, but for now the change here is to just use the freshly-built stub instead of pulling it from the installed location.